src/Components/Atbash/AtbashInput.js

```diff
@@ -1,72 +1,126 @@
-                    c = String.fromCharCode(atbashCode);
-                }
-            }
-            strOut += c;
-        }
-        console.log("strOut is: " + strOut + "\n");
-        this.setState({
-            outputStr: strOut
-        })
-    }
+class Atbash extends Component {
+  state = {
+    inputStr: null,
+    outputStr: null,
+  };
+  handleChange = (e) => {
+    this.setState({
+      [e.target.id]: e.target.value,
+    });
+  };
+  handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(this.state);
 
-    backwardsAlpha= () => {
-        var i;
-        let letters = ""
-        for(i = 25; i >= 0; i--) {
-            letters+=String.fromCharCode(i+65);
+    var strOut = "";
+    var str = this.state.inputStr;
+    var atbashCode = 0;
+
+    for (var i = 0; i < str.length; i++) {
+      var c = str[i];
+      if (c.match(/[a-z]/i)) {
+        let code = str.charCodeAt(i);
+        if (code >= 65 && code <= 90) {
+          code = code - 65;
+          atbashCode = ((26 - code + 25) % 26) + 65;
+          c = String.fromCharCode(atbashCode);
+        }
+        // Lowercase letters
+        else if (code >= 97 && code <= 122) {
+          code = code - 97;
+          atbashCode = ((26 - code + 25) % 26) + 97;
+          c = String.fromCharCode(atbashCode);
         }
-        console.log(letters)
-        return letters
-    } 
+      }
+      strOut += c;
+    }
+    console.log("strOut is: " + strOut + "\n");
+    this.setState({
+      outputStr: strOut,
+    });
+  };
 
-    render(){
-        return(
-            <div className="container" id="atbash_cipher">
-            <p className="title">ATBASH CIPHER: </p>
-                <LetterBox default={true} offset={0}></LetterBox>
-                <LetterBox default={false} offset={0} length={17} letters={this.backwardsAlpha()}></LetterBox>
-                <form onSubmit={this.handleSubmit}>
-                    <label htmlFor="input">Input your plain text: </label>
-                    <input type="text" id="inputStr" onChange={this.handleChange} />
-                    <button>Submit</button>
-                </form>
-                <p>Your ciphered Atbash text is: {this.state.outputStr}</p>
-            </div>
-        )
+  backwardsAlpha = () => {
+    var i;
+    let letters = "";
+    for (i = 25; i >= 0; i--) {
+      letters += String.fromCharCode(i + 65);
     }
+    console.log(letters);
+    return letters;
+  };
+
+  render() {
+    return (
+      <div className="container">
+        <div className="vertical-spacing">
+          <p className="subtitle">
+            So in Caesar cipher, each letter in the alphabet turned into the
+            letter x places after it.
+          </p>
+          <LetterBox default={true} length={11} offset={0}></LetterBox>
+          <div classname="row">
+            <Arrow />
+            <Arrow />
+            <Arrow />
+            <Arrow />
+            <Arrow />
```

Comment:
You had mentioned to me that you were having issues with the arrows not lining up with the elements you want. The trick here is that you might want to create a component that draws an arrow between two components that you specify on the page, or make the arrows accommodate changes in viewport dimensions. You can do this by making the component render an SVG whose height and width change with the viewport.